### PR TITLE
Updating russian strings

### PR DESCRIPTION
**Pull request by Matroftt**
**Describe what the proposed change does**
- Fixes some mistakes
- Translates strings that were added in newer commits

### CODE REVIEW

File: CorsixTH/Lua/languages/russian.lua [hunk 1]
```diff
@@ -61,10 +64,15 @@ new_game_window = {
...
+  caption = "Дополнительные кампании",
+  start_selected_campaign = "Начать кампанию",
+}
 menu_options_warmth_colors = {
-  choice_2 = "  СИНИЙ ЗЕЛЕНЫЙ КРАСНЫЙ  ",
-  choice_1 = "  КРАСНЫЙ  ",
-  choice_3 = "  ЖЕЛТЫЙ ОРАНЖЕВЫЙ КРАСНЫЙ  ",
+  choice_2 = "  Синий, зеленый, красный ",
+  choice_1 = "  Красный  ",
+  choice_3 = "  Желтый, оранжевый, красный  ",
```
Comment (lewri) [inline CorsixTH/Lua/languages/russian.lua:75]: I think some menu items are intentionally capitalised, but I don't have the documentation to support it. Do we all caps in other languages?

Comment (Matroftt) [reply CorsixTH/Lua/languages/russian.lua:75]: > I think some menu items are intentionally capitalised, but I don't have the documentation to support it. Do we all caps in other languages?

Yes in other languages there is caps, but due to option menu was already not using caps in russian.lua, i decided to keep it like that. Also i would notice that Arial (which is a default font for unicode languages) looks... well, not good while being capitalised in a menu.

In this PR
image
In CorsixTH v0.67
image

Comment (lewri) [reply CorsixTH/Lua/languages/russian.lua:75]: Could you try testing out GoNotoKurrent-Bold.ttf as your Unicode font linked in #2398 ? It'd be interesting how readable russian becomes.

Comment (Matroftt) [reply CorsixTH/Lua/languages/russian.lua:75]: > Could you try testing out GoNotoKurrent-Bold.ttf as your Unicode font linked in #2398 ? It'd be interesting how readable russian becomes.

It's alright but the text is being a little scattered and sometimes goes beyond of dialog borders

image

Comment (lewri) [reply CorsixTH/Lua/languages/russian.lua:75]: That's usually expected with a font change annoyingly. Does GoNotoKurrent-Regular look better (also in that PR I think)?

**Comment by Matroftt** [comment]
Ready to merge

**Comment by Alberth289346** [comment]
You may want to remove the "[WIP]" prefix in the title (assuming you added that).

**Comment by Matroftt** [comment]
> You may want to remove the "[WIP]" prefix in the title (assuming you added that).

My bad

**Pull request merged**